### ISSUE TITLE
High CPU usage

### ISSUE THREAD

**Issue by Omar301293** [engintron/engintron#1383]
First, I am not an expert in operating systems, but I always try to learn and follow all the existing explanations for your own program. I have used it for a long time and it is very good and I recommend it to everyone,

In the morning I found a message that there was an update, I updated like any other good person 😂 , but after the update there is a very large and cramping use of my site and I do not know where the problem is from, the previous version was really great and I did not face any problems with it

Is it possible to know why this happened after the update?

**Comment by fevangelou** [comment]
The Nginx config is unchanged compred to v1.16.0. Perhaps the update triggered something else in your system or its coincidental, especially if it's a VPS server...

Assuming you already have Munin installed on your server, what does it report under the "system" graphs (for CPU usage)?

**Comment by VR51** [comment]
Hope you don't mind me chiming in. Is it possible Engintron has not installed fully? The update to v1.16.0 should be performed via Terminal. The info is in the changelog https://engintron.com/docs/#/pages/Changelog.

Am I right in thinking that memcached and APCu need to be reinstalled after an Engintron update, or does this apply only after am OS, Apache and PHP update?

I usually clear the error logs using the Engintron GUI then restart my servers after an update when I notice the update spawned high resource usage then I review the new errors when CPU, memory or MySQL spikes persist.

I only dropped by here to say thank you, @fevangelou, for this update. I wondered why Tuning Primer and MySQL Tuner were not part of the Engintron GUI. I'm glad to see you have added them. I thought the same about memcached and APCu; maybe this update will surprise me here too.

Hope the above helps. I'm crawling back into my servers again now.... Happy New Year!

**Comment by fevangelou** [comment]
@VR51 Thank you for your kind words. I actually tested more than 10 public servers updating to v2 via WHM and there was no issue with the upgrade whatsoever. So I doubt this is the issue. As for the installers, they have been updated, but you don't need to run them again. Only if you update your EasyApache4 setup.

**Comment by adam-mazzei** [comment]
I also noticed an increase in server load after I upgraded to version 2.
I didn't encounter any errors during the installation but even so, I completely removed Engintron and reinstalled it from the command line. And the load gets higher with Engintron active. When deactivating, the load drops immediately. Munin's System graph shows a considerable increase after the update in the 2nd week of the year, see image:

Vultr VPS 2Core - 4Gb Ram

![Captura de tela 2022-01-31 181242](https://user-images.githubusercontent.com/5430044/151873859-679a084a-7789-4623-b63d-4381563dd01d.jpg)

**Comment by fevangelou** [comment]
I can't tell why this has happened for you. Unless there is a traffic pattern change on your server... Engintron's Nginx config is the same for v2 as it was for the latest release in the 1.x series. So what you describe is not really applicable here...
